### ISSUE TITLE
 8 bit "Mcufriend" shield and LVGL

### ISSUE THREAD

**Issue by mltongca** [Bodmer/TFT_eSPI#1995]
Following your instructions to set up 8 bit "Mcufriend" shield on ESP32 UNO style board with a resistance based touch screen, the display and the touch screen work very well. Is it possible to for this display and touch screen to work with LVGL? Thanks.

**Comment by Bodmer** [comment]
Yes, this will work but I do not have an example for that library. Consult the LVGL documentation.
